pom.xml

```diff
@@ -25,16 +25,18 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <java.version>1.8</java.version>
-    <spring-boot.version>1.5.2.RELEASE</spring-boot.version>
+    <spring-boot.version>1.5.9.RELEASE</spring-boot.version>
```

Comment:
could you remove this property if it is not being used?

Reply:
Yes, will do, 1 min...

Reply:
Done